azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/service/impl/deploy/RunFromZipFunctionDeployHandler.java

```diff
@@ -8,9 +8,19 @@
 
 import java.io.File;
 
-public class RunFromZipFunctionDeployHandler  implements IFunctionDeployHandler{
+public class RunFromZipFunctionDeployHandler extends ZIPFunctionDeployHandler {
+    private static final String APP_SETTING_WEBSITE_RUN_FROM_PACKAGE = "WEBSITE_RUN_FROM_PACKAGE";
+    private static final String RUN_FROM_PACKAGE_VALUE = "1";
+
     @Override
     public void deploy(File file, WebAppBase functionApp) {
-
+        DeployUtils.updateFunctionAppSetting(functionApp, APP_SETTING_WEBSITE_RUN_FROM_PACKAGE, RUN_FROM_PACKAGE_VALUE);
+        try {
+            // work around for issue https://dev.azure.com/msazure/Unified%20Platform%20KPIs/_workitems/edit/7481871
+            Thread.sleep(20 * 1000);
```

Comment:
must be 20 seconds?

Reply:
Recommanded values from function service team